x-pack/plugins/cases/server/common/models/case_with_comments.ts

```diff
@@ -160,7 +160,6 @@ export class CaseCommentModel {
           updated_at: date,
           updated_by: { ...this.params.user },
         },
-        version: this.caseInfo.version,
```

Comment:
Removing the version we tell the SO client to not throw on conflicts. It will retry automatically.